src/main/java/se/sundsvall/supportmanagement/service/NotificationService.java

```diff
@@ -52,16 +53,19 @@ public List<Notification> getNotifications(final String municipalityId, final St
 
 	public String createNotification(final String municipalityId, final String namespace, final Notification notification) {
 
-		if (!isOwner(notification.getOwnerId()) && doesNotificationExist(municipalityId, namespace, notification)) {
+		if (notification.getOwnerId() == null || isExecutingUserTheOwner(notification.getOwnerId()) || doesNotificationExist(municipalityId, namespace, notification)) {
 			return null;
 		}
 
 		final var entity = NotificationMapper.toNotificationEntity(namespace, municipalityId, notification);
+
 		return notificationRepository.save(entity).getId();
 	}
 
-	private boolean isOwner(final String ownerId) {
-		return Objects.equals(ownerId, executingUserSupplier.getAdUser());
+	private boolean isExecutingUserTheOwner(final String ownerId) {
+		return Optional.ofNullable(ownerId)
+			.map(id -> Objects.equals(id, executingUserSupplier.getAdUser()))
+			.orElse(false);
 	}
```

Comment:
Is this necessary? It makes a null check for ownerId and then returns Objects.equals(ownerId, executingUserSupplier.getAdUser()). But before the method is called, there is already a null check.

Also, if ownerId is null, in the first predicate, it returns true. But if it is null in this predicate, it returns false.